wordpress-mobile/WordPress-Android
# [Blogging Prompts Enhancements] Turn feature flag on

Fixes #17855

Make the default value of the `BloggingPromptsEnhancementsFeatureConfig` `true`. Also creates a remote field for it, so it can be turned off remotely if needed.

**Note: This is being merged bypassing the branch protections because of [instability in BuildKite](https://www.buildkitestatus.com/incidents/k9nh8yy0px4q) CI reporting.**

The build ran successfully:
![image](https://user-images.githubusercontent.com/5091503/216728574-f7ccac57-3383-488b-90e3-8f3efb722398.png)


To test:
1. Uninstall Jetpack (to make sure the local settings are cleared)
2. Install Jetpack from this build
3. Login and select a blogging site
4. Go to `Debug Settings`
5. **Verify** the `blogging_prompts_enhancements_enabled` flag is on by default

You can also go ahead and check that the enhancements features are on as well. That is:
- Enable / Disable settings for Prompts
- Choose your own prompt from Prompts list
- "Turn off" menu item in prompts card

## Regression Notes
1. Potential unintended areas of impact
The view more answers feature being turned on.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing.

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
